include/fluidsynth/midi.h

```diff
@@ -58,6 +58,15 @@ extern "C" {
 typedef int (*handle_midi_event_func_t)(void *data, fluid_midi_event_t *event);
 /* @} */
 
+/**
+ * Generic callback function fired once by MIDI tick change.
+ *
+ * @param data User defined data pointer
+ * @param tick The current (zero-based) tick, which triggered the callback
+ * @return Should return #FLUID_OK on success, #FLUID_FAILED otherwise
+ */
+typedef int (*handle_midi_tick_func_t)(void *data, int tick);
```

Comment:
Please elaborate a little bit on this documentation by giving an example, why/when this tick-callback could be useful. Doing might help users to better understand when to use which callback type.

Reply:
And it would avoid that some future day a maintainer would say: "Hey, what's that? We don't need that, strip it away." ;)

Reply:
done :)